apps/docs/content/components/navigation/Link.mdx

```diff
@@ -4,6 +4,7 @@ description: A link allows a user to navigate to a different location.
 category: "navigation"
 links:
     source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/Link/src/Link.tsx
+    aria: https://www.w3.org/WAI/ARIA/apg/patterns/link
```

Comment:
just a nitpick thing, some links end in a slash and others don't. maybe we can pick one and clean this up?

Reply:
I'd follow the URL's that are displayed when navigating the w3 org website.